packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -366,11 +420,15 @@ export interface BaseSetting {
     /** List of words to always be considered incorrect. */
     flagWords?: string[];
 
-    /** List of words to be ignored. An Ignored word will not show up as an error even if it is also in the `flagWords`. */
+    /**
+     * List of words to be ignored. An ignored word will not show up as an error, even if it is
+     * also in the `flagWords`.
+     */
     ignoreWords?: string[];
 
     /**
-     * True to enable compound word checking.
+     * True to enable compound word checking. See [the documentation](https://cspell.org/docs/configuration/case-and-accent-sensitivity/) for more details.
```

Comment:
The current link is: `[Case Sensitivity](https://cspell.org/docs/case-sensitive/)`

Reply:
can you offer this as a suggestion so i can click the button "commit suggestion"?